src/Microsoft.Extensions.DiagnosticAdapter/Resources.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="Converter_MethodTypeMustBeInterface" xml:space="preserve">
+    <value>Type '{0}' on method '{2}' must be an interface in order to support proxy generation from source type '{1}'.</value>
+  </data>
```

Comment:
Generally what you'd want to do here is say "Unable to generate a proxy for method 'foo'. See InnerException for details" and then let the inner exception explain it.